src/main/java/com/application/views/masterdetail/SkigebietForm.java

```diff
@@ -0,0 +1,170 @@
+import com.vaadin.flow.component.Key;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.button.ButtonVariant;
+import com.vaadin.flow.component.formlayout.FormLayout;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.textfield.BigDecimalField;
+import com.vaadin.flow.component.textfield.IntegerField;
+import com.vaadin.flow.component.textfield.NumberField;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.data.binder.BeanValidationBinder;
+import com.vaadin.flow.data.binder.Binder;
+import com.vaadin.flow.data.binder.ValidationException;
+import com.vaadin.flow.shared.Registration;
+
+
+public class SkigebietForm extends FormLayout{
+
+    Binder<SkiResort> binder = new BeanValidationBinder<>(SkiResort.class);
+    TextField name =new TextField("Name");
+    TextField region = new TextField("Region");
+    TextField operator = new TextField("Betreiber");
+    TextField address = new TextField("Addresse");
+    IntegerField zip = new IntegerField("Postleitzahl");
+    TextField city = new TextField("Stadt");
+    IntegerField heightMin = new IntegerField("min Höhenlage");
+    IntegerField heightMax = new IntegerField("max Höhenlage");
+    IntegerField totalLenght = new IntegerField("Pistenlänge");
+    IntegerField ropeways = new IntegerField("Anzahl Pisten");
+    NumberField pos_lon = new NumberField("Laengengrad");
+    NumberField pos_lat = new NumberField("Breitengrad");
+    TextField dateSeasonStart = new TextField("Saisonstart");
+    TextField dateSeasonEnd = new TextField("Saisonende");
+    TextField timeServiceStart = new TextField("Öffnungszeiten");
+    TextField timeServiceEnd = new TextField("Schließzeiten");
+    IntegerField currentUtilizationPercent = new IntegerField("Auslastung");
+    IntegerField userRating = new IntegerField("User Bewertungen");
+    NumberField weatherCurrentWindspeed = new NumberField("aktuelle Windgeschwindigkeit");
+    NumberField weatherCurrentTemperature = new NumberField("aktuelle Temperatur");
+    IntegerField weatherCurrentSnowfallForecastPercent = new IntegerField("Schneefall Forecast in %");
+    IntegerField weatherCurrentSnowfallForecastAmountMM = new IntegerField("Schneefall Forecast in mm");
+    TextField weatherDatetimeLastRead = new TextField("Letzte Eingabe Wetter");
+    IntegerField snowDepthMin = new IntegerField("Schneehöhe min ");
+    IntegerField snowDepthMax = new IntegerField("Schneehöhe max");
+    IntegerField amountFreshSnow = new IntegerField("Neuschneehöhe");
+    TextField dateLastSnowfall = new TextField("Datum letzter Schneefall");
+    TextField URLTicketpage = new TextField("Ticketpreise");
+    IntegerField avalancheWarningLevel = new IntegerField("aktuelle Warnsufe");
+    TextField URLImageFront = new TextField("Titelbild");
+    TextField URLImageSlope = new TextField("Pistenbild");
+
+    Button save = new Button("Save");
```

Comment:
Die für den User ersichtlichen Benennungen Save/Cancel/Delete in deutsch hinschreiben wie vereinbart (Codequalität)
Variablen => Englisch // UI=> Deutsch

Reply:
erldeigt